src/mod_muc_room.erl

```diff
@@ -156,9 +156,14 @@ init([Host, ServerHost, Access, Room, HistorySize, RoomShaper, Opts, QueueType])
     add_to_log(room_existence, started, NewState),
     {ok, normal_state, NewState}.
 
+%% This is a top level call to normal_state to kill the muc room during any event.
+%% Another hack we should remove once we have the process that sweeps empty mucs
+normal_state(Event, StateData) ->
+	{_, _, NewStateData} = normal_state(Event, StateData, true),
+	close_room_without_occupants(NewStateData).
 normal_state({route, <<"">>,
 	      #message{from = From, type = Type, lang = Lang} = Packet},
-	     StateData) ->
+	     StateData, _) ->
```

Comment:
What's the `_` mean?

Reply:
It means ignore this function parameter.  Its _real_ hacky, but we'll rip it out.